src/value_classes/value_id.rs

```diff
@@ -265,6 +265,81 @@ impl ValueID {
         }
     }
 
+    pub fn set_bool(&self, value: bool) -> bool {
+        match self.get_type() {
+            ValueType::ValueType_Bool | ValueType::ValueType_Button => {
+                let manager_ptr = unsafe { extern_manager::get() };
+                unsafe { extern_manager::set_value_bool(manager_ptr, &self.as_ozw_vid(), value) }
+            }
+            _ => false
+        }
+    }
+
+    pub fn set_byte(&self, value: u8) -> bool {
+        if self.get_type() == ValueType::ValueType_Byte {
+            let manager_ptr = unsafe { extern_manager::get() };
+            unsafe { extern_manager::set_value_byte(manager_ptr, &self.as_ozw_vid(), value) }
+        } else {
+            false
+        }
+    }
+
+    pub fn set_float(&self, value: f32) -> bool {
+        if self.get_type() == ValueType::ValueType_Decimal {
+            let manager_ptr = unsafe { extern_manager::get() };
+            unsafe { extern_manager::set_value_float(manager_ptr, &self.as_ozw_vid(), value) }
+        } else {
+            false
+        }
+    }
+
+    pub fn set_int(&self, value: i32) -> bool {
+        if self.get_type() == ValueType::ValueType_Int {
+            let manager_ptr = unsafe { extern_manager::get() };
+            unsafe { extern_manager::set_value_int(manager_ptr, &self.as_ozw_vid(), value) }
+        } else {
+            false
+        }
+    }
+
+    pub fn set_short(&self, value: i16) -> bool {
+        if self.get_type() == ValueType::ValueType_Short {
+            let manager_ptr = unsafe { extern_manager::get() };
+            unsafe { extern_manager::set_value_short(manager_ptr, &self.as_ozw_vid(), value) }
+        } else {
+            false
+        }
+    }
+
+    pub fn set_string(&self, value: &str) -> bool {
+        // The underlying C++ lib accepts strings for all types
+        let manager_ptr = unsafe { extern_manager::get() };
+        if let Ok(c_string) = CString::new(value) {
+            unsafe { extern_manager::set_value_string(manager_ptr, &self.as_ozw_vid(), c_string.as_ptr()) }
+        } else {
+            false
+        }
+    }
+
+    pub fn set_raw(&self, value: &Vec<u8>) -> bool {
+        if self.get_type() == ValueType::ValueType_Raw && value.len() < 256 {
+            let manager_ptr = unsafe { extern_manager::get() };
+            unsafe { extern_manager::set_value_raw(manager_ptr, &self.as_ozw_vid(), value.as_ptr(), value.len() as u8) }
+        } else {
+            false
+        }
+    }
+
+    pub fn set_list_selection_string(&self, value: &str) -> bool {
+        if self.get_type() == ValueType::ValueType_List {
+            if let Ok(c_string) = CString::new(value) {
+                let manager_ptr = unsafe { extern_manager::get() };
+                return unsafe { extern_manager::set_value_list_selection_string(manager_ptr, &self.as_ozw_vid(), c_string.as_ptr()) };
+            }
+        }
+        false
+    }
```

Comment:
I'd prefer to returns `Result<(), ()>` instead of bool, this makes it easier to use with `try!` etc.